### PR TITLE
Added micro-AFH

### PR DESCRIPTION
**Pull request by Jukspa**
Not sure if this should be handled as another jumptype instead

### CODE REVIEW

File: spt/strafestuff.cpp [hunk 1]
```diff
@@ -41,6 +41,9 @@ ConVar tas_strafe_version("tas_strafe_version",
                           FCVAR_TAS_RESET,
                           "Strafe version. For backwards compatibility with old scripts.");
 
+ConVar tas_strafe_afh_length("tas_strafe_afh_length", "0.0000000000000000001", FCVAR_TAS_RESET, "Magnitude of AFHs");
```
Comment (YaLTeR) [inline spt/strafestuff.cpp:44]: Is this something you can actually input? Also, is this cvar needed? Maybe just hardcode the value

File: spt/strafestuff.cpp [hunk 2]
```diff
@@ -1028,7 +1042,7 @@ namespace Strafe
 	{
 		if (StrafeJump(jumped, player, vars, ducking, out))
 		{
-			if (!yawChanged || tas_strafe_allow_jump_override.GetBool())
+			if ((!yawChanged || tas_strafe_allow_jump_override.GetBool()) && !out.Processed)
```
Comment (YaLTeR) [inline spt/strafestuff.cpp:1045]: Is this change related?

File: spt/strafestuff.cpp (hunk 1 above)
Comment (Jukspa) [reply spt/strafestuff.cpp:44]: yes and I figured I'd leave it in case there's a problem with the default value of the cvar not working for whatever reason in some scenario

File: spt/strafestuff.cpp (hunk 2 above)
Comment (Jukspa) [reply spt/strafestuff.cpp:1045]: the other jump commands run the MapSpeeds function that sets the fmove and smove for that frame so that needs to be skipped with this method

Comment (Jukspa) [reply spt/strafestuff.cpp:1045]: Would you prefer if this got refactored into the StrafeJump function instead?

File: spt/strafestuff.cpp (hunk 1 above)
Comment (Jukspa) [reply spt/strafestuff.cpp:44]: I had some instances where it would work for some jumps and not for others and I'm not really sure why

Comment (YaLTeR) [reply spt/strafestuff.cpp:44]: Oh, okay. I mean, it is pretty small, so there's a good chance that it underflows to 0 on some operation.

File: spt/strafestuff.cpp (hunk 2 above)
Comment (YaLTeR) [reply spt/strafestuff.cpp:1045]: I see. Idk, whatever you think is better.

Comment (Jukspa) [reply spt/strafestuff.cpp:1045]: I think I'll refactor it

**Comment by Jukspa** [comment]
done
